monitoring/metrics.rb

```diff
@@ -0,0 +1,249 @@
+require "google/cloud/monitoring"
+
+# Avoid collisions with other runs
+RANDOM_SUFFIX = Time.now.to_i.to_s
```

Comment:
Better to use an actual pseudo-random number. This won't avoid collisions; a second is a long time. Perhaps:

```ruby
RANDOM_SUFFIX = rand(36**10).to_s(36)
```

But also see below comments.

Reply:
Removed RANDOM_SUFFIX